search-app/docs/api.md

```diff
@@ -1186,6 +1188,14 @@ Find collections that match all of the 'project' param values
 
    curl "%CMR-ENDPOINT%/collections?created_at\[\]=2000-01-01T10:00:00Z,2010-03-10T12:00:00Z&created_at\[\]=2015-01-01T10:00:00Z,"
 
+#### <a name="c-with-new-granules"></a> Find collections with new granules
+
+  This supports option `and`.
+
+  Find collections containing granules added within the range of datetimes. The datetime has to be in yyyy-MM-ddTHH:mm:ssZ format. The default is inclusive on the range boundaries.
+
+  curl "%CMR_ENDPOINT%/collections?has_granules_created_at=\[\]2015-01-01T10:00:00Z"
```

Comment:
Looks like this is now a range - should there be a trailing `,` to indicate no end date?